src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -403,17 +402,24 @@ default String metadataWebhook() {
     }
 
     @ConfigItem(
-        keyName = "ignoreSeasonalWorlds",
-        name = "Ignore Seasonal Worlds",
+        keyName = "seasonalPolicy",
+        name = "Seasonal Policy",
         description = "Whether to suppress notifications that occur on seasonal worlds like Leagues.<br/>" +
             "Note: the Leagues-specific notifier uses an independent config toggle",
         position = 1015,
         section = advancedSection
     )
-    default boolean ignoreSeasonal() {
-        return false;
+    default SeasonalPolicy seasonalPolicy() {
+        return SeasonalPolicy.ACCEPT;
```

Comment:
could also consider making this `FORWARD_TO_LEAGUES` by default (for new plugin installs)

Reply:
& when leagues override is not set, it just uses primary url